docs/Containers/WireGuard.md

```diff
@@ -103,9 +103,60 @@ With most containers, you can continue to tweak environment variables and settin
 
 #### <a name="configurePeerDNS"> Optional configuration - DNS resolution for peers </a>
 
-* `PEERDNS=`. The default value of `auto` instructs the WireGuard container to use the same DNS as other containers when resolving requests from connected peers. In practice, that means the container directs queries to 127.0.0.11, which Docker intercepts and forwards to whichever resolvers are specified in the Raspberry Pi's `/etc/resolv.conf`.
+You have several options for how your remote peers resolve DNS requests:
+
+* `PEERDNS=auto`
+
+	The default value of `auto` instructs the WireGuard *service* running within the WireGuard *container* to use the same DNS as the WireGuard *container* when resolving requests from connected peers. In practice, that means the *service* directs queries to 127.0.0.11, which Docker intercepts and forwards to whichever resolvers are specified in the Raspberry Pi's `/etc/resolv.conf`.
+
+* <a name="customContInit"> `PEERDNS=auto` with `custom-cont-init` </a>
+
+	This configuration instructs WireGuard to forward DNS queries from remote peers to any **container** which is listening on port 53. This is the option you will want to choose if you are running an ad-blocking DNS server (eg *PiHole* or *AdGuardHome*) in a container on the same host as WireGuard, and you want your remote clients to obtain DNS resolution via the ad-blocker.
```

Comment:
Please add something like:
And you don't want the pi host to use the pihole or adguardhome. If you have done the resolveconf.conf changes from the pihole docs, wireguard will out-of-the-box use any container providing dns service, and the custom-cont-init isn't needed.